TestCases/coupled_cht/solid_solid_contact_resistance/fluid.cfg

```diff
@@ -0,0 +1,55 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: solid-to-solid and solid-to-fluid conjugate heat transfer  %
```

Comment:
Didn't you add the configs to the tutorials repo? Or just the meshes?

Reply:
The configs, meshes, and gmesh .geo scripts are in the tutorials repo. I'll remove the contact resistance test case from TestCases, as the tutorial contains all that's required for running the regression test.